app/AboutScreen.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
```

Comment:
Is this screen used at all?

Reply:
I just got it started and figured we could expand on it later.